mintlify/mint
# Mintlify dev

## Summary
Sets up the mintlify dev CLI. Allows you to run your local instance of your docs by running `mintlify dev` at the directory where your `mint.json` file lives. It will auto-update and listen to changes, so you don't have to refresh the client.
Details
- Refactors the CLI to use [yargs](https://www.npmjs.com/package/yargs)
- Uses [Ora](https://www.npmjs.com/package/ora) for loading
- File listener using [chokidar](https://www.npmjs.com/package/chokidar)

## Test Plan
1. `cd packages/mintlify`
2. `npm install`
3. `npm run build`
4. `npm run local`
5. Navigate to any of our customers' `docs` folder
6. Run `mintlify dev`
It should run their docs locally.
